### ISSUE TITLE
0.47.2: Minimap spawns offscreen when closed

### ISSUE THREAD

**Issue by Windchild292** [MegaMek/megamek#1672]
This is from Discord:
"
any time after i launch and close megamek in which i had the minimap open and visible on screen, on the second load in it will be spawned off screen. usually to the right of my rightmost screen.
"

**Comment by mkerensky** [comment]
Yeah this still happens to me too, on a brand new build of WIndows 10 as well as my two previous Windows 7 installations. If I change my primary monitor in Windows to the other screen, the minimap appears, I can move it and then set my screen settings back to normal.  The workaround will last until I exit MM.

**Comment by TenkawaBC** [comment]
There is now a reset function to reset window positions.

Multi-monitor setups and remote desktop sessions can really mess with stored window positions.
